Gemfile

```diff
@@ -272,7 +272,7 @@ group :test do
   gem "selenium-devtools"
   gem "selenium-webdriver", "~> 4.20"
 
-  gem "fuubar", "~> 2.5.0"
+  gem "fuubar", "~> 2.5.0", require: false
```

Comment:
For faster boot times?

Reply:
Yes. It does not save much, but it's still an easy win.